moshi/src/test/java/com/squareup/moshi/JsonAdapterTest.java

```diff
@@ -114,7 +114,7 @@
       fail();
     } catch (JsonDataException expected) {
       assertThat(expected).hasMessage("Unexpected null at $[1]");
-      assertThat(reader.nextNull()).isNull();
+      assertThat(reader.<Object>nextNull()).isNull();
```

Comment:
interesting: why do these tests require updating when moving to Gradle?

Reply:
whoops, meant to revert this. Was my early indicator that I needed to set the language version to 7 😅

Reply:
62bf821